<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendManager</class>
 <widget class="QWidget" name="FriendManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FriendManager{
	background-color:white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetManagerBtn" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:5px;
	border:none;
	border:1px solid rgb(196,196,196);
}

QPushButton:hover{
	background-color:rgb(245,245,245);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>好友管理器</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetMsgBtn" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetMsgBtn{
	border-bottom:1px solid rgb(246,246,246);
}
QPushButton{
	border:none;
	background:transparent;
}
QPushButton:hover{
	background-color:rgb(245,245,245);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnFriendMsg">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>好友通知                                &gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGroupMsg">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>群通知                                   &gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetChoose" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btnGroup{
	border:none;
	border-radius:10px;
	background-color:transparent;
	color:rgb(127,127,127);
}

QPushButton#btnFriend{
	border:none;
	border-radius:10px;
	background-color:transparent;
	color:rgb(0,153,255);
}

QPushButton:hover{
	color:rgb(190,190,190);
}

QWidget#widgetChooseBtn{
	background-color:rgb(245,245,245);
	border-radius:10px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QWidget" name="widgetChooseBtn" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnFriend">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>好友</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGroup">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>群聊</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetTree" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidgetTree">
        <property name="styleSheet">
         <string notr="true">QWidget{
	border:none;
}
QTreeWidget{
	background:transparent;
	border:none;
}
QTreeWidget::item{
	border:none;
	background:transparent;
}
QTreeWidget::item:hover{
	border:none;
	background:transparent;
}
QTreeWidget::item:focused{
	border:none;
	background:transparent;
}
QTreeWidget::item:selected{
	border:none;
	background:transparent;
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageFriend">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="treeWidgetFriend">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QScrollBar:vertical{
	background-color: transparent;
	width:10px;
	margin:0px;
	border-radius:5px;
}
QScrollBar::handle:vertical{
	background-color: rgb(215,215,215);
	border-radius:5px;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical{
	height:0px;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
	background: transparent;
}</string>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageGroup">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="treeWidgetGroup">
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
